index.md

```diff
@@ -11,13 +11,12 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
 
 **Training:**
 
-* [CodeSchool: Try Git][]
-* [CodeSchool: Git Real 2][]
-* [CodeSchool: Git Real][]
-* [Sparkbox how_to][]: your first experience with Git
-* [More Git][]: more Git practice
-* [CodeSchool: Try Ruby][]
-* [Ruby setup][]: set up your machine for running Ruby
+* ~~[CodeSchool: Try Git][]~~
```

Comment:
Ah, I see what happened.  Hey @laurendorman, love that you're using the curriculum to track your progress, but we probably shouldn't push it to this remote as happened in a0bb350.  I'll show you how to remove that commit tomorrow.